tests/test_slicing.py

```diff
@@ -14,9 +14,7 @@
 
 
 def test_find_exits():
-    slicing_test = angr.Project(os.path.join(test_location, 'x86_64', 'cfg_1'),
-                                use_sim_procedures=True,
-                                default_analysis_mode='symbolic')
+    slicing_test = angr.Project(os.path.join(test_location, 'x86_64', 'cfg_1'),use_sim_procedures=True, default_analysis_mode='symbolic', auto_load_libs=False)
```

Comment:
Please keep these arguments on multiple lines and append rather than reformatting.

Reply:
Changes done. please review.

Reply:
Does not appear to be, also note that this applies to the other cases in this file